spec/tapioca/compilers/dsl/active_job_spec.rb

```diff
@@ -99,10 +99,10 @@ def perform(user_id)
 
           class NotifyJob
             class << self
-              sig { params(user_id: Integer).returns(T.any(NotifyJob, FalseClass)) }
+              sig { params(user_id: ::Integer).returns(T.any(NotifyJob, FalseClass)) }
```

Comment:
The types in `T.any(NotifyJob, FalseClass)` don't get a `::` since that string is hard-coded by the DSL generator: https://github.com/Shopify/tapioca/blob/main/lib/tapioca/compilers/dsl/active_job.rb#L55

Might be worth updating the API to generate `RBI::` types to use real sorbet types and then convert them to strings to things like the NamePatch would apply. Alternatively these DSL generators could be updated to hard-code the `::` or use `::Tapioca::Reflection`.